backend/structs.py

```diff
@@ -94,11 +95,12 @@ def get_id(self) -> str:
 
     # end login required methods
 
-    def set_password(self, password: str) -> None:
-        self.password = generate_password_hash(password)
+    def set_password(self, password: Optional[str]) -> None:
```

Comment:
why is the password optional?

Reply:
huh good point.  i dont remember and dont have a good reason.

I thinkkkkkk because the test was putting in empty passwords if thats not a correct reason haha